### PR TITLE
Add the global arguments to the cli reference

### PR DESCRIPTION
**Pull request by macneale4**

### CODE REVIEW

**Review by timsehn — COMMENTED** [review]
I might make this a Global arguments section just under this section. Like so:

\# Global arguments

The reason the CLI is there is to give the impression that this is Git immediately.

**Comment by macneale4** [comment]
I don't understand your feedback. Are you saying you want the top of the doc to be formatted like git's help? git's help lists the global args before anything else:

```
$ git --help
usage: git [-v | --version] [-h | --help] [-C <path>] [-c <name>=<value>]
           [--exec-path[=<path>]] [--html-path] [--man-path] [--info-path]
           [-p | --paginate | -P | --no-pager] [--no-replace-objects] [--bare]
           [--git-dir=<path>] [--work-tree=<path>] [--namespace=<name>]
           [--super-prefix=<path>] [--config-env=<name>=<envvar>]
           <command> [<args>]

These are common Git commands used in various situations:

start a working area (see also: git help tutorial)
   clone     Clone a repository into a new directory
   init      Create an empty Git repository or reinitialize an existing one

work on the current change (see also: git help everyday)
   add       Add file contents to the index
[snip]
```

File: content/reference/cli.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ title: CLI
 # CLI
 
 ```
-$ dolt
+$ dolt --help
```
Comment (timsehn) [inline content/reference/cli.md]:
```suggestion
$ dolt
```

File: content/reference/cli.md [hunk 2]
```diff
@@ -49,6 +49,27 @@ Valid commands for dolt are
                 dump - Export all tables in the working set into a file.
                 docs - Commands for working with Dolt documents.
                stash - Stash the changes in a dirty working directory away.
+
```
Comment (timsehn) [inline content/reference/cli.md:55]:
```suggestion
# Global Arguments

```

File: content/reference/cli.md [hunk 3]
```diff
@@ -49,6 +49,27 @@ Valid commands for dolt are
...
+Dolt subcommands are in transition to using the flags listed below as global flags.
```
Comment (timsehn) [inline content/reference/cli.md:56]: Add a code bloc starter here.

**Review by timsehn — COMMENTED** [review]
Added comments to make my desire clearer

**Comment by timsehn** [comment]
You can also commit it and I can edit it. Or I can make a PR against this one.

**Comment by macneale4** [comment]
Oh, I see what you are saying. I'll tweak.

**Comment by macneale4** [comment]
I'm shipped because risk is low!

**Pull request merged**